### ISSUE TITLE
multiprocess for input plugin "forward"

### ISSUE THREAD

**Issue by mperriere** [fluent/fluentd#777]
Hello,
first a quick overview of our Fluentd utilization:
we have 40 Apache front server producing 1tb log per day. Those logs are send to 2 fluentd relay then send to an Amazon Kinesis repo.
We noticed a performance bottleneck with a 5Mb/s. We our 40 HTTP nodes, we should be able to handle at least 40Mb/s.

We managed to reach good performances by using @type multiprocess with 20 nodes:
cat td-agent.conf

```
<source>
  @type multiprocess
  <process>
    cmdline -c /etc/td-agent/td-agent-child1.conf --log /var/log/td-agent/td-agentd_child1.log
    sleep_before_start 1s
    sleep_before_shutdown 5s
    log_level debug
  </process>
  <process>
    cmdline -c /etc/td-agent/td-agent-child2.conf --log /var/log/td-agent/td-agentd_child2.log
    sleep_before_start 2s
    sleep_before_shutdown 5s
    log_level debug
  </process>
  <process>
    cmdline -c /etc/td-agent/td-agent-child3.conf --log /var/log/td-agent/td-agentd_child3.log
    sleep_before_start 3s
    sleep_before_shutdown 5s
    log_level debug
  </process>
  <process>
    cmdline -c /etc/td-agent/td-agent-child4.conf --log /var/log/td-agent/td-agentd_child4.log
    sleep_before_start 4s
    sleep_before_shutdown 5s
    log_level debug
  </process>
  <process>
    cmdline -c /etc/td-agent/td-agent-child5.conf --log /var/log/td-agent/td-agentd_child5.log
    sleep_before_start 5s
    sleep_before_shutdown 5s
    log_level debug
  </process>
</source>
```

Each of the 5 childs handle 4 HTTP nodes.
But the configuration becomes quite messy.
The best solution for us to keep a maintanable configuration set would be an integrated input forward multi process.
Without this feature, Fluentd is not considered safe for my company to process our HTTP log stream. Kafka or a similar tool will be used instead.

Is multiprocess for forward plugin planned for the next release?
Thank you
Mathieu Perriere.

**Comment by repeatedly** [comment]
> Is multiprocess for forward plugin planned for the next release?

This feature will be included in the v0.14 release, not v0.12 series.
v0.14 is now under development so we don't have stable release recently.

**Comment by repeatedly** [comment]
This is listed in v0.14 list.
https://github.com/fluent/fluentd/issues/829
